dm/master/config.go

```diff
@@ -50,13 +50,15 @@ var SampleConfigFile string
 // NewConfig creates a config for dm-master
 func NewConfig() *Config {
 	cfg := &Config{}
+	cfg.Debug = false
```

Comment:
It sames that `cfg.Debug` is useless now. We can delete it in next PR.

Reply:
some more useless code needs to be deleted later.